src/Bundle/Storage/SessionStorage.php

```diff
@@ -13,7 +13,9 @@
 
 namespace Sylius\Bundle\ResourceBundle\Storage;
 
+use Sylius\Component\Resource\Exception\StorageUnavailableException;
```

Comment:
Since there is ongoing initiative to move namespaces in 1.11, new exception should go directly into target namespace.
 
```suggestion
use Sylius\Resource\Exception\StorageUnavailableException;
```